packages/prisma/schema.prisma

```diff
@@ -436,6 +436,7 @@ model Team {
   activeOrgWorkflows     WorkflowsOnTeams[]
   attributes             Attribute[]
   smsLockReviewedByAdmin Boolean                 @default(false)
+  features               Feature[]               @relation("team_features")
```

Comment:
Why are we using snake casing on these?

Reply:
Named to be consistent with user_eventtypes that we already have

Reply:
Isn't the snake casing legacy and we should aim to have pascal casing moving forward?

Reply:
adressed on latest changes